src/components/GlobalStyle/GlobalStyle.js

```diff
@@ -0,0 +1,39 @@
+import { createGlobalStyle } from 'styled-components';
+
+import { fontWeightNormal, fontWeightBold, fontMedium, colorWhite, colorDarkBlue } from '../../styles/designTokens';
+
+const GlobalStyle = createGlobalStyle`
+  html {
+    font-size: 62.5%;
+  }
+
+  body {
+    margin: 0;
+    padding: 0;
+    font-family: Tahoma, sans-serif;
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
```

Comment:
I believe that in `styled-components` they are using [`stylis`](https://github.com/thysultan/stylis.js) so it is doing autoprefixing